### ISSUE TITLE
Redistributing package with GAP

### ISSUE THREAD

**Issue by alex-konovalov** [gap-system/SingularInterface#58]
How far is this package from being submitted for the redistribution with GAP? Is that feasible that it can appear in GAP 4.8 and maybe supersede the [Singular](http://www.gap-system.org/Packages/singular.html) package? I'd like to encourage the authors considering this option. If it will be redistributed with GAP, it will be used in regression tests to check its compatibility with the core system and other packages, which may help to discover situations when new changes in them will break your package. It also helps to know which functionality of the core system is used in packages - otherwise, some things may be changed or even remove on the assumption that none of the packages uses them, and of course we have no way to look into all undeposited packages...

**Comment by fingolfin** [comment]
I do not think it is ready for delievery without somebody investing additional work.

Indeed, my feeling is that this package is still very crash prone. I think it would be a bad idea to let it loose on people in this state, as bad impressions stick, and once we are known to be "that package that always crashes", it would be hard to get people to change their mind on that later on.

At this point, the problem is not getting people to test things -- I know lots of things that are broken, and can easily generate problem cases with little effort.

No, the actual problem is that nobody seems to have time to work on this, and hence no progress is made. E.g. myself, I know lots of things I could do to improve this package -- but none of them are of mathemtical nature, they are all pure engineering problems. The cynic view is: I won't be hired on for doing any of that work... :-/. So my motivation is low. And I am not aware of anybod else actively working on this project.

**Comment by alex-konovalov** [comment]
@fingolfin thanks. Okay, the outcome of this is that someone (probably me) would have to make minor update the [Singular package](http://www.gap-system.org/Packages/singular.html) to make it GAP 4.8-compatible. I don't mind, but had a hope that this may be avoided ;-)
